test/test_Times.cc

```diff
@@ -0,0 +1,71 @@
+#include "gtest/gtest.h"
+#include <assert.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <math.h>
+#include "../generic.h"
+#include "../SW_Sky.h"
+#include "../filefuncs.h"
+#include "../myMemory.h"
+#include "../SW_Defines.h"
+#include "../SW_Files.h"
+#include "../SW_Model.h"
+#include "../SW_Site.h"
+#include "../SW_SoilWater.h"
+#include "../SW_VegProd.h"
+#include "../SW_Site.h"
+#include "../SW_Flow_lib.h"
+#include "../Times.h"
+#include "sw_testhelpers.h"
+
+namespace{
+  // Test the 'Times.c' function 'interpolate_monthlyValues'
+  TEST(TimesTest, interpolateMonthlyValues){
+    // point to the structure that contains cloud coverage monthly values
+    SW_SKY SW_Sky;
+    SW_SKY *interpolate = &SW_Sky;
+    unsigned int i;
+    // set all monthlyValues all 10
+    for (i = 0; i < length(interpolate -> cloudcov); i++){
+      interpolate -> cloudcov[i] = 10;
+    }
+    interpolate -> cloudcov_daily[0] = 0;
```

Comment:
Why do you have to specifically so cloudcov_daily[0] to 0 here? Should this be part of the functionality.

Reply:
I did that to illustrate that cloudcov_daily[0] does not change by setting it before the function call than testing that is still that value afterwords. The function does not modify this value so it is whatever it is before the call.